src/templates/contacts.html.j2

```diff
@@ -36,12 +37,16 @@ OSG Contacts
           {%- endif %}
         </p>
       </div>
-      {% if user.GitHub -%}
       <div class="row">
         <div class="col-sm-3"><span class="font-weight-bold">GitHub:</span></div>
-        <div class="col-sm"><a href="https://github.com/{{user.GitHub}}">{{ user.GitHub }}</a></div>
+        <div class="col-sm">
+          {% if user.GitHub -%}
+          <a href="https://github.com/{{user.GitHub}}">{{ user.GitHub }}</a>
+          {%- else -%}
+          <span class="text-danger">(none)</span>
```

Comment:
Maybe make it explicit.  Like have the `(none)` text be:

    (none) Update github account name here

Or something more clever?  Bonus points if you make it a mailto:help@opensciencegrid.org with subject and some body already filled out.

Reply:
That might be a little too glaring; we don't have GitHub info for 99% of our contacts.